frontend/apps/filemanager/filemanager.lua

```diff
@@ -157,6 +157,30 @@ function FileManager:init()
                         UIManager:close(self.file_dialog)
                     end,
                 },
+                {
+                    text = _("Purge .sdr"),
+                    enabled = lfs.attributes(util.realpath(file), "mode") == "file" and util.realpath(file):match("%.") and
+                            lfs.attributes(util.realpath(file):match("(.*)%.")..".sdr", "mode") == "directory",
```

Comment:
The problem with getSidecarDir is that, as it is, it will fail when filepath does not containt a dot : 
doc_path = "/mnt/onboard"; print(doc_path:match("(.*)%.")..".sdr") => attempt to concatenate a nil value
It may not fail in all these parts because i checked before 1) it is not a directory 2) it does contain a dot (which may be not needed if we _assume_ filemanager only show files with extensions it knows, so only file with dots - but that's a lot of precautions and assumptions to do before calling getSidecarDir ; I thought it would be as well to put the whole code here.
May be better would be to put in DocSettings a new function : hasSidecarDir, that would do all these checks there ?